webapp/src/components/modals/update_run_status_modal.tsx

```diff
@@ -133,33 +153,53 @@ const UpdateRunStatusModal = ({
         }
     };
 
+    const description = () => {
+        const broadcastChannelCount = run?.broadcast_channel_ids.length ?? 0;
+        const followersChannelCount = runMetadata?.followers?.length ?? 0;
+
+        const OverviewLink = (...chunks: string[]) => (
+            <Link to={pluginUrl(`/runs/${playbookRunId}/overview`)}>
+                {chunks}
+            </Link>
+        );
+
+        if ((broadcastChannelCount + followersChannelCount) === 0) {
+            return formatMessage({
+                defaultMessage: 'This update will be saved to <OverviewLink>overview page</OverviewLink> and broadcasted to <OverviewLink>0 channels</OverviewLink>',
+            }, {OverviewLink});
+        }
+
+        return formatMessage({
+            defaultMessage: 'This update will be broadcasted to {hasBroadcast, select, true {<OverviewLink><ChannelsTooltip>{broadcastChannelCount, plural, =1 {one channel} other {{broadcastChannelCount, number} channels}}</ChannelsTooltip></OverviewLink>} other {}}{hasFollowersAndChannels, select, true { and } other {}}{hasFollowers, select, true {<OverviewLink><FollowersTooltip>{followersChannelCount, plural, =1 {one direct message} other {{followersChannelCount, number} direct messages}}</FollowersTooltip></OverviewLink>} other {}}.',
+        }, {
+            OverviewLink,
+            ChannelsTooltip: (...chunks) => (
+                <Tooltip
+                    id={`${ID}_broadcast_channels_tooltip`}
+                    content={generateTooltipText(broadcastChannelNames, broadcastChannelCount)}
+                >
+                    <span tabIndex={0}>{chunks}</span>
+                </Tooltip>
+            ),
+            FollowersTooltip: (...chunks) => (
+                <Tooltip
+                    id={`${ID}_broadcast_followers_tooltip`}
+                    content={generateTooltipText(followerNames, followersChannelCount)}
+                >
+                    <span tabIndex={0}>{chunks}</span>
+                </Tooltip>
+            ),
+            hasFollowersAndChannels: Boolean(broadcastChannelCount && followersChannelCount).toString(),
+            hasBroadcast: Boolean(broadcastChannelCount).toString(),
+            hasFollowers: Boolean(followersChannelCount).toString(),
```

Comment:
Super nit: we use here "Channels" and "Broadcast". Can we use the same for both? For example:
```suggestion
            hasFollowersAndChannels: Boolean(broadcastChannelCount && followersChannelCount).toString(),
            hasChannels: Boolean(broadcastChannelCount).toString(),
            hasFollowers: Boolean(followersChannelCount).toString(),
```